mailchimp3/baseapi.py

```diff
@@ -42,16 +42,28 @@ def _iterate(self, url, **kwargs):
         kwargs['count'] = integer
         kwargs['offset'] = integer
         """
+        #fields as a kwarg ought to be a string with comma-separated substring
+        #values to pass along to self._mc_client._get(). it also ought to
+        #contain 'total_items' in all cases, so that is enforced here
+        if kwargs.get('fields') is None:
+            kwargs['fields'] = 'total_items'
+        else:  # assume well formed string, just missing 'total_items'
+            if not 'total_items' in kwargs['fields'].split(','):
```

Comment:
As employed in the examples: `results = client.lists.members.all('123456', get_all=True, fields='members.ids,members.email_address')`

Of course, there's no reason we could not favor a list of string values instead of ',' joined string values. I would probably prefer that tbh, but it conflicts with examples.